src/custom/hooks/useSwapCallback.ts

```diff
@@ -55,19 +56,31 @@ async function _postOrder(params: PostOrderParams): Promise<string> {
   const { inputAmount, outputAmount } = trade
 
   const path = trade.route.path
-  const selToken = path[0]
+  const sellToken = path[0]
   const buyToken = path[path.length - 1]
+  const sellAmount = inputAmount.raw.toString(10)
+  const buyAmount = outputAmount.raw.toString(10)
+
+  // TODO: This might disappear, and just take the state from the state after the fees PRs are merged
+  //  we assume, the solvers will try to satisfy the price, and this fee is just a minimal fee.
+  // Get Fee
+  const { feeRatio, minimalFee } = await getFeeInformation(chainId, sellToken.address)
```

Comment:
Yes, this should already be prefilled by fee middleware

Reply:
If I understand the order correctly. Fee is added onto the order before posting but after user already sees the numbers without the fee.
That's not good

Reply:
This is an intermediate PR for the DEMO

I'm not 100% sure we have the calculations are right for the other PR. This PRs just walk in the direction of adding the fee. When the other PR is ready, we'll be able to put the two together

In the DEMO they will probably return 0 anyways

Reply:
i think this is a placeholder, otherwise agree